packages/fether-react/src/utils/withHealth.js

```diff
@@ -16,38 +16,36 @@ import {
   switchMap,
   take
 } from 'rxjs/operators';
-import isElectron from 'is-electron';
 import isEqual from 'lodash/isEqual';
 import { peerCount$, syncStatus$, withoutLoading } from '@parity/light.js';
 
 import parityStore from '../stores/parityStore';
 
-const electron = isElectron() ? window.require('electron') : null;
+// The preload scripts injects `ipcRenderer` into `window.bridge`
+const { ipcRenderer, isParityRunningStatus } = window.bridge;
 
 const isApiConnected$ = parityStore.isApiConnected$;
 
 const isParityRunning$ = Observable.create(observer => {
-  if (electron) {
-    electron.ipcRenderer.on('parity-running', (_, isParityRunning) => {
+  if (ipcRenderer) {
+    ipcRenderer.on('parity-running', (_, isParityRunning) => {
       observer.next(isParityRunning);
     });
   }
-}).pipe(
-  startWith(electron ? !!electron.remote.getGlobal('isParityRunning') : false)
-);
+}).pipe(startWith(isParityRunningStatus ? !!isParityRunningStatus : false));
 
 const downloadProgress$ = Observable.create(observer => {
-  if (electron) {
-    electron.ipcRenderer.on('parity-download-progress', (_, progress) => {
+  if (ipcRenderer) {
+    ipcRenderer.on('parity-download-progress', (_, progress) => {
       observer.next(progress);
     });
   }
 }).pipe(startWith(0));
 
 const isClockSync$ = Observable.create(observer => {
-  if (electron) {
-    electron.ipcRenderer.send('asynchronous-message', 'check-clock-sync');
-    electron.ipcRenderer.once('check-clock-sync-reply', (_, clockSync) => {
+  if (ipcRenderer) {
```

Comment:
not related to this PR but I think we can safely remove all the "is electron"-type conditions cc @amaurymartiny ? or do we continue to support loading fether inside the browser ?

Reply:
+1, we can drop this